### DISCUSSION TITLE
Stability issue with Athom LB01-15W-E27-TAS bulbs and Tasmota 11.0.0

### DISCUSSION THREAD

**Discussion by R4D4R** [arendst/Tasmota#15258] [Q&A]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): __LB01-15W-E27-TAS___
- [x] Tasmota binary firmware version number used: _11.0.0____
  - [x] Pre-compiled
  - [ ] Self-compiled
- [x] Flashing tools used: _OTA____
- [x] Provide the output of command: `Backlog Template; Module; GPIO 255`: 
23:10:21.425 CMD: Backlog Template; Module; GPIO 255
23:10:21.510 MQT: stat/tasmota_D2FB0E/RESULT = {"NAME":"LB01-15W-E27-TAS","GPIO":[0,0,0,0,416,419,0,0,417,452,418,0,0,0],"FLAG":0,"BASE":18}
23:10:21.720 MQT: stat/tasmota_D2FB0E/RESULT = {"Module":{"0":"LB01-15W-E27-TAS"}}
23:10:21.974 MQT: stat/tasmota_D2FB0E/RESULT = {"GPIO0":{"0":"None"},"GPIO1":{"0":"None"},"GPIO2":{"0":"None"},"GPIO3":{"0":"None"},"GPIO4":{"416":"PWM1"},"GPIO5":{"419":"PWM4"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"417":"PWM2"},"GPIO13":{"452":"PWM_i5"},"GPIO14":{"418":"PWM3"},"GPIO15":{"0":"None"},"GPIO16":{"0":"None"},"GPIO17":{"0":"None"}}
```lua
  Configuration output here: N/A
  

```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here: N/A, none in use


```
- [x] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here: 23:11:37.648 CMD: Status 0
23:11:37.656 MQT: stat/tasmota_D2FB0E/STATUS = {"Status":{"Module":0,"DeviceName":"Bathroom Track 5","FriendlyName":["Bathroom Track 5"],"Topic":"tasmota_D2FB0E","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0}}
23:11:37.663 MQT: stat/tasmota_D2FB0E/STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota.bin.gz","RestartReason":"Power On","Uptime":"0T02:44:17","StartupUTC":"2022-03-29T19:27:20","Sleep":50,"CfgHolder":4617,"BootCount":38,"BCResetTime":"2021-08-19T01:45:17","SaveCount":2089,"SaveAddress":"F6000"}}
23:11:37.673 MQT: stat/tasmota_D2FB0E/STATUS2 = {"StatusFWR":{"Version":"9.3.1(tasmota)","BuildDateTime":"2021-03-09T16:12:28","Boot":31,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"422/699"}}
23:11:37.680 MQT: stat/tasmota_D2FB0E/STATUS3 = {"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["<redacted>",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00208009","0405C80001000600003C5A0A000000000000","08000080","00006400","00004000"]}}
23:11:37.691 MQT: stat/tasmota_D2FB0E/STATUS4 = {"StatusMEM":{"ProgramSize":593,"Free":408,"Heap":25,"ProgramFlashSize":1024,"FlashSize":2048,"FlashChipId":"1540A1","FlashFrequency":40,"FlashMode":3,"Features":["00000809","8FDAC787","04368001","000000CF","010013C0","C000F981","00004004","00001000","00000000"],"Drivers":"1,2,3,4,5,6,7,8,9,10,12,16,18,19,20,21,22,24,26,27,29,30,35,37,45","Sensors":"1,2,3,4,5,6"}}
23:11:37.698 MQT: stat/tasmota_D2FB0E/STATUS5 = {"StatusNET":{"Hostname":"tasmota_D2FB0E-6926","IPAddress":"192.168.10.172","Gateway":"192.168.10.1","Subnetmask":"255.255.255.0","DNSServer":"8.8.4.4","Mac":"C4:4F:33:D2:FB:0E","Webserver":2,"WifiConfig":4,"WifiPower":17.0}}
23:11:37.704 MQT: stat/tasmota_D2FB0E/STATUS6 = {"StatusMQT":{"MqttHost":"192.168.10.32","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_D2FB0E","MqttUser":"<redacted>","MqttCount":1,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30}}
23:11:37.712 MQT: stat/tasmota_D2FB0E/STATUS7 = {"StatusTIM":{"UTC":"2022-03-29T22:11:37","Local":"2022-03-29T23:11:37","StartDST":"2022-03-27T02:00:00","EndDST":"2022-10-30T03:00:00","Timezone":"+01:00","Sunrise":"06:33","Sunset":"19:15"}}
23:11:37.720 MQT: stat/tasmota_D2FB0E/STATUS10 = {"StatusSNS":{"Time":"2022-03-29T23:11:37"}}
23:11:37.729 MQT: stat/tasmota_D2FB0E/STATUS11 = {"StatusSTS":{"Time":"2022-03-29T23:11:37","Uptime":"0T02:44:17","UptimeSec":9857,"Heap":25,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"POWER":"OFF","Dimmer":90,"Color":"000000C620","HSBColor":"317,71,0","White":90,"CT":200,"Channel":[0,0,0,78,13],"Scheme":4,"Fade":"OFF","Speed":1,"LedTable":"ON","Wifi":{"AP":1,"SSId":"<redacted>","BSSId":"48:8F:5A:54:D7:53","Channel":1,"RSSI":62,"Signal":-69,"LinkCount":1,"Downtime":"0T00:00:03"}}}


```
- [x] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here: 23:13:04.508 CMD: weblog 4
23:13:04.515 MQT: stat/tasmota_D2FB0E/RESULT = {"WebLog":4}
23:13:04.720 WIF: Checking connection...
23:13:05.528 CFG: Saved to flash at F5, Count 2090, Bytes 4096
23:13:22.383 WIF: Sending Gratuitous ARP
23:13:24.726 WIF: Checking connection...
23:13:44.702 WIF: Checking connection...
23:15:04.929 WIF: Checking connection...

```

### TO REPRODUCE
_Steps to reproduce the behavior:_ Leave power on to bulb, turn on/off bulb, wait random amount of time (usually 1-24hr).

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._  Light turns on and off

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._
I have 6 of these bulbs in a track light. Every day since upgrading to 11.0.0 (about 1 month ago) I had to turn the light switch off/on 2-3 times a day because 1 or 2 or 3 bulbs would stop responding to wifi.

All 6 bulbs are within 10 ft of an access point. When this occurs, my AP stops receiving DHCP requests from the bulbs. DHCP lease time didn't seem to matter - Set to 24h, some bulbs would stop communicating part way through the lease period. Set to 15m, some bulbs would not request a new lease. The bulb that would drop out appeared to be random - I kept track of which ones were dropping out and couldn't find a pattern.

A few points which may or may not be noteworthy:

-This seems to occur more often when the bulbs are turned on or already on. I've only experienced 1 time where the bulb did not turn on, the other times the bulb simply wouldn't turn off.
-A power cycle temporarily fixes the issue
-Wifi is via a Mikrotik hAP Lite configured as an AP. WPA2, "aes ccm" for Unicast and Group ciphers. Signal DB is from -62 dBm to -68 dBm.
-Bulbs were originally taken out of the box, connected to wifi, upgraded to latest Tasmota, then connected to HA via MQTT. I don't recall setting an SetOption values (but it's been a while since I set these up).
-I have 1x LB01-15W-E27-TAS bulb in a different part of the house running on 10.1.0 and has not experienced this issue.
-I have 1x LB01-15W-E27-TAS-1 bulb on 11.0.0 and it has not experienced this issue.
-I rolled back to Tasmota 9.3.1 on these 6 bulbs and have not experienced this issue since (been almost a week now).
-Another user on Reddit is having a very similar issue that may or may not be related: https://www.reddit.com/r/tasmota/comments/tjf8v4/athom_15w_1400lm_rgbcct_bulb_lb0115we27_dimmer/

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by smcgann99** [comment]
I'm having exactly the same issues with these bulbs. I'm downgrading 1 of them to 10.0.0 to see if it solves my problems too.
